pkg/cli/context.go

```diff
@@ -305,7 +305,9 @@ var sqlCtx = struct {
 	// demoCluster is the interface to the in-memory cluster for the
 	// `demo` command, if that is the command being run.
 	demoCluster democluster.DemoCluster
-}{}
+}
+
+var sqlCtx = sqlContext{}
```

Comment:
it looks like we still need this global, your commit message makes it sound like it goes away permanently (which i understand it doesn't). i guess my question is how are we enforcing noone typos `c.sqlCtx` instead of `sqlCtx` in future?

Reply:
Because the next PR #67462 actually moves the file away into a separate package, where the global simply does not exist any more.